tests/assembly/asm/x86-types.rs

```diff
@@ -315,99 +315,99 @@ macro_rules! check_reg {
 
 // CHECK-LABEL: reg_i16:
 // CHECK: #APP
-// x86_64: mov r{{[a-z0-9]+}}, r{{[a-z0-9]+}}
-// i686: mov e{{[a-z0-9]+}}, e{{[a-z0-9]+}}
+// CHECK-X86_64: mov r{{[a-z0-9]+}}, r{{[a-z0-9]+}}
+// CHECK-I686: mov e{{[a-z0-9]+}}, e{{[a-z0-9]+}}
 // CHECK: #NO_APP
 check!(reg_i16 i16 reg "mov");
 
 // CHECK-LABEL: reg_f16:
 // CHECK: #APP
-// x86_64: mov r{{[a-z0-9]+}}, r{{[a-z0-9]+}}
-// i686: mov e{{[a-z0-9]+}}, e{{[a-z0-9]+}}
+// CHECK-X86_64: mov r{{[a-z0-9]+}}, r{{[a-z0-9]+}}
+// CHECK-I686: mov e{{[a-z0-9]+}}, e{{[a-z0-9]+}}
```

Comment:
Hmm. I feel like I would prefer `CHECK-x86_64` and `CHECK-i686`? i.e. exact match of rev name.

Reply:
Uppercasing was some of the point of this since it matches builtin directives, and is what LLVM uses ([209 completely lowercase](https://github.com/search?q=repo%3Allvm%2Fllvm-project+%2F%28%3F-i%29check-prefix%5B%3D+%5D%5Ba-z0-9_-%5D%2B%2F&type=code) vs. [24.4k completely uppercase](https://github.com/search?q=repo%3Allvm%2Fllvm-project+%2F%28%3F-i%29check-prefix%5B%3D+%5D%5BA-Z0-9_-%5D%2B%2F&type=code), mixed < 1k). So I think it is some of what people are used to, which I think is why we seem to have a mix of uppercase and lowercase revision names.

(plus, at least in my editor (helix), it only highlights if the string leading up to the `:` is all uppercase, so it looks more consistent. Obviously that is the about the furthest thing from a good reason to do anything, and I have no idea about other editors, but bonuses are nice).

Reply:
Sure. I can get used to it, probably, the other concern about ordering of the rev and CHECK would rate higher if any.

Reply:
I like `X86_64` or `CHECK-x86_64`. When I encounter unfamiliar terms, all-uppercase is difficult to understand.

Reply:
Agreed with the revision name exact matching. I'm not sure if in compiletest we currently treat revision names as case-insensitive (if we don't, then we probably should), but IMO an exact case match against a revision name is less confusing. For target names especially, sometimes the casing contributes to more familiarity.

I don't have a *strong* opinion on this stylistically, anyway.

Reply:
> I like `X86_64` or `CHECK-x86_64`. When I encounter unfamiliar terms, all-uppercase is difficult to understand.

I would counter that it is more likely for someone newish to understand that `CHECK-X86_64` pertains to x86_64, than it is for someone newish to understand that `X86_64:` is indicating a pattern that gets checked.